lib/Mage/System/Ftp.php

```diff
@@ -51,8 +51,8 @@ protected function checkConnected()
     /**
      * ftp_mkdir wrapper
      *
-     * @param stin$name
-     * @return unknown_type
+     * @param string $name
+     * @return string
```

Comment:
https://www.php.net/manual/en/function.ftp-mkdir.php
> Returns the newly created directory name on success or false on error. 

```suggestion
    * @return string|false
```